tests/integration/components/article-content-test.js

```diff
@@ -90,11 +90,6 @@ module('Integration | Component | article content', (hooks) => {
       }}{{/article-content}}`);
 
     assert.dom(topLeaderboardSelector).exists({ count: 1 });
-    assert.equal(
-      find(topLeaderboardSelector).previousSibling,
-      find('.wiki-page-header'),
-      'previous element is site header',
-    );
```

Comment:
What's wrong with this test?

Reply:
Thanks for catching. It was removed at the time when top-leaderboard had additional wrapper. It's no longer the case. Assertion reverted.